streampark-flink/streampark-flink-client/streampark-flink-client-core/src/main/scala/org/apache/streampark/flink/client/impl/KubernetesNativeApplicationClient.scala

```diff
@@ -62,14 +67,16 @@ object KubernetesNativeApplicationClient extends KubernetesNativeClientTrait {
     var clusterDescriptor: KubernetesClusterDescriptor = null
```

Comment:
remove this code

Reply:
done

Reply:
@Al-assad Hello, thank you for your review, I think this piece is deleted again when the code is finished